### PR TITLE
Fixes an issue in the AccountPage.js file

### PR DESCRIPTION
**Pull request by salvare736**
## Description

Fixes an issue where, on the My Account page, submitting an edit to the account details would not properly reflect the changes made to the currently logged in account (the bug would instead display the data for llama010)

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/AccountPage/AccountPage.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const AccountPage = props => {
     axiosWithAuth()
       .put(`/profile/${updatedUserInfo.user_id}`, updatedUserInfo)
       .then(res => {
-        setHrfUserInfo(res.data.updated_profile);
+        setHrfUserInfo(updatedUserInfo);
       })
       .catch(err => console.log(err));
```
Comment (elCarlosSantiago) [inline src/components/pages/AccountPage/AccountPage.js:50]: Do we want to log the err for the client? Could be helpful to keep for now while we're developing.

**Review by elCarlosSantiago — APPROVED** [review]
Great Job! One small comment in the review. Thanks!

File: src/components/pages/AccountPage/AccountPage.js (hunk 1 above)
Comment (salvare736) [reply src/components/pages/AccountPage/AccountPage.js:50]: Good point. We may need to go through at some point to remove console logs like these. But like you said, they are helpful during development.

**Comment by n0tybynature** [comment]
Code looks good to me.

**Review by JamesWhite1911 — APPROVED** [review]
Looks good to me I'll go ahead and approve

**Pull request merged**